site/get-started.pug

```diff
@@ -47,39 +47,47 @@ html(lang='en-US' dir="ltr").spectrum.spectrum--light.spectrum--medium
+                p.spectrum-Body.spectrum-Body--sizeL We definitely need the #[code.spectrum-Code.spectrum-Code--sizeS button] package in order to add a button to our project. We're going to use the #[code.spectrum-Code.spectrum-Code--sizeS page] component to wrap our button in some default styles. The #[code.spectrum-Code.spectrum-Code--sizeS vars] and #[code.spectrum-Code.spectrum-Code--sizeS tokens] packages provide global Spectrum CSS styling for responsiveness and themes that we will need for our components to display properly.
+                
+                p.spectrum-Body.spectrum-Body--sizeL Installed components will be available in the node_modules/@spectrum-css/ folder.
 
-                h3.spectrum-Heading.spectrum-Heading--sizeS Setting up global variables
+                h3.spectrum-Heading.spectrum-Heading--sizeS Using components
                 p.spectrum-Body.spectrum-Body--sizeL Create a new HTML file (e.g., #[code.spectrum-Code.spectrum-Code--sizeS button.html]) in your project’s root folder.
 
-                p.spectrum-Body.spectrum-Body--sizeL or displaying a button in Spectrum’s light color theme and medium scale, a couple of classes need to be added to the document’s #[code.spectrum-Code.spectrum-Code--sizeS &lt;html&gt;] tag:
+                p.spectrum-Body.spectrum-Body--sizeL Spectrum CSS components utilize custom properties in order to change themes and scales. For these to apply, a couple of classes need to be added to the document’s #[code.spectrum-Code.spectrum-Code--sizeS &lt;html&gt;] tag based on the scale and theme you wish to use. For example, the following code snippet will display styling for the medium scale and light color theme:
 
                 pre.spectrum-CodeBlock
                   code.spectrum-Code.spectrum-Code--sizeS.language-markup!=util.Prism.highlight('<html class="spectrum spectrum--medium spectrum--light">', util.Prism.languages.markup, 'markup')
 
-                h3.spectrum-Heading.spectrum-Heading--sizeS Adding Spectrum CSS
-                p.spectrum-Body.spectrum-Body--sizeL Inside the #[code.spectrum-Code.spectrum-Code--sizeS &lt;head&gt;] tag, include the stylesheets that contain the design system tokens for global variables, scales, and color themes, as well as the stylesheets for the components themselves:
+                p.spectrum-Body.spectrum-Body--sizeL Inside the #[code.spectrum-Code.spectrum-Code--sizeS &lt;head&gt;] tag, include the stylesheets that contain the design system tokens for global variables, scales, and color themes, as well as the stylesheets for the button and page components:
 
                 -
                   htmlMarkup = util.Prism.highlight(`<head>
+                    <!-- Include global variables first -->
                     <link rel="stylesheet" href="node_modules/@spectrum-css/vars/dist/spectrum-global.css">
```

Comment:
Doesn't the button need `@spectrum-css/tokens` package and not the `@spectrum-css/vars` package? It's included in the `npm install` above, but I don't see it used anywhere.

Reply:
Yep you're right that was just something I forgot to remove. @pfulton I know we're still in the process of migrating a few components but are you ok with removing the vars package from the instructions and only including tokens?

Reply:
Yeah, it might be time to go Spectrum Tokens-only—we're very close to being completely switched over. @castastrophe what do you think?

We could also hold this for a little bit until the migration work is complete.